internal/provider/azure.go

```diff
@@ -93,3 +96,36 @@ func (p *AzureTrialInputProvider) region() string {
 	}
 	return DefaultAzureRegion
 }
+
+func (p *AzureLiteInputProvider) Provide() Values {
```

Comment:
The same question as defined here: https://github.com/kyma-project/kyma-environment-broker/pull/976/files#r1700033908